sensor_msgs/PointCloud2 cloud
geometry_msgs/Point center
geometry_msgs/Point[] voxel_points
geometry_msgs/Vector3[] voxel_normals
rl_msgs/SegmentationFace[] faces
sensor_msgs/RegionOfInterest region
